anndata/_io/zarr.py

```diff
@@ -190,9 +190,12 @@ def write_csr(f, key, value: sparse.csr_matrix, dataset_kwargs=MappingProxyType(
     group.attrs["shape"] = value.shape
     write_array(group, "data", value.data, dataset_kwargs=dataset_kwargs)
     write_array(group, "indices", value.indices, dataset_kwargs=dataset_kwargs)
+    if "chunks" in dataset_kwargs:
+        del dataset_kwargs["chunks"]
```

Comment:
Wouldn't this mutate the `dataset_kwargs` `dict` and change downstream behavior?

Reply:
I am not sure how deep "API stability" goes, but it seemed the only place this could come from was [here](https://github.com/theislab/anndata/pull/568/files#diff-3cbce68b63ac7913ef30291e65e6d8fe881c362e361b5206ed1651d688ab3d70R48-R49) so where it is set explicitly.